firefox_puppeteer/ui/about_window/window.py

```diff
@@ -0,0 +1,140 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime
+
+from marionette_driver import By, Wait
+
+from ..windows import BaseWindow
+from ...api.software_update import SoftwareUpdate
+from deck import Deck
+
+
+class AboutWindow(BaseWindow):
+    """Representation of the About window."""
+
```

Comment:
Please add the two DTDs which contain necessary labels of elements and which are referenced here:
https://dxr.mozilla.org/mozilla-central/source/browser/base/content/aboutDialog.xul#11

Properties we don't have. But for both add a test as we do for the other window types in the basic test.